ESP32_AP-Flasher/wwwroot/main.js

```diff
@@ -1419,11 +1423,13 @@ async function getTagtype(hwtype) {
 			options: Object.values(jsonData.options ?? []),
 			zlib: parseInt(jsonData.zlib_compression || "0", 16),
 			shortlut: parseInt(jsonData.shortlut),
-			busy: false
+			busy: false,
+			usetemplate:parseInt(jsonData.usetemplate || "0",16)
```

Comment:
usetemplate is defined in the json as a decimal integer, not a hex (yes, that's a bit inconvenient when creating a tagtype definition, but it's a bit too much to change it and have it backwards compatible)